server/database/seed.js

```diff
@@ -8,7 +9,15 @@ const userdescriptions = ['Globetrotter', 'Avid Traveller', 'Novice Nomad'];
 
 const userages = [1, 2, 3, 4];
 
-const seedDB = () => {};
+const seedDB = () => {
```

Comment:
I like how you used callback functions here to insure that everything gets seeded in order!